docs/user/03-40-debug-function.md

```diff
@@ -0,0 +1,94 @@
+# Debug a Function
+
+This tutorial shows how to use an external IDE to debug a Function in Kyma CLI.
+
+## Steps
+Follows these steps:
+
+<div tabs name="steps" group="debug-function">
+  <details>
+  <summary label="vsc_node">
+  Visual Studio Code (Node.js)
+  </summary>
+
+1. In VSC, navigate to the location of the file with the Function definition.
+2. Create the `.vscode` directory.
+3. In the `.vscode` directory, create the `launch.json` file with this content:
+   ```json
+   {
+     "version": "0.2.0",
+     "configurations": [
+       {
+         "name": "attach",
+         "type": "node",
+         "request": "attach",
+         "port": 9229,
+         "address": "localhost",
+         "localRoot": "${workspaceFolder}/kubeless",
+         "remoteRoot": "/kubeless",
+         "restart": true,
+         "protocol": "inspector",
+         "timeout": 1000
+       }
+     ]
+   }
+    ```
+4. Run the Function with the `--debug` flag.
+    ```bash
+    kyma run function --debug
+    ```
+
+</details>
+<details>
+<summary label="vsc_python">
+Visual Studio Code (Python)
+</summary>
+
+1. In VSC, navigate to the location of the file with the Function definition.
+2. Create the `.vscode` directory.
+3. In the `.vscode` directory, create the `launch.json` file with this content:
+   ```json
+   {
+      "version": "0.2.0",
+      "configurations": [
+          {
+              "name": "Python: Kyma function",
```

Comment:
Yes. Should I change it?

Reply:
I wonder if we could arrange the content more elegantly so that identical text is needed just once, but I have no specific suggestion for that.
Like, just one instruction for VS code, and only in step 3, differentiate between "for node.js, use this code (...) and for python, that code (...)"